apps/ledger-live-mobile/src/screens/Onboarding/steps/useCaseSelection.tsx

```diff
@@ -37,34 +36,34 @@ const OnboardingStepUseCaseSelection = () => {
   const getProductName = (modelId: DeviceModelId) =>
     getDeviceModel(modelId)?.productName.replace("Ledger", "").trimStart() || modelId;
 
-  const onCloseProtectDrawer = useCallback(() => {
+  const onCloseProtectDrawer = () => {
     setIsProtectDrawerOpen(false);
-  }, []);
+  };
 
-  const onBuyNanoX = useCallback(() => {
+  const onBuyNanoX = () => {
     Linking.openURL("https://shop.ledger.com/pages/ledger-nano-x");
-  }, []);
+  };
 
-  const onDiscoverBenefits = useCallback(() => {
+  const onDiscoverBenefits = () => {
     Linking.openURL("http://ledger.com");
-  }, []);
+  };
 
-  const onPressNew = useCallback(() => {
+  const onPressNew = () => {
     dispatch(setOnboardingType(OnboardingType.setupNew));
     navigation.navigate(ScreenName.OnboardingModalSetupNewDevice, {
       deviceModelId,
     });
-  }, [deviceModelId, dispatch, navigation]);
```

Comment:
Does `deviceModelId` updates every time you select a device and you click on?

Reply:
Yes it does